src/frontend/apps/standalone_site/src/routes/routes.tsx

```diff
@@ -185,6 +194,9 @@ export const routes: Routes = {
         aria-label="svg-menu-my-playlists"
       />
     ),
+    subRoutes: {
+      UPDATE_PLAYLIST: { path: '/my-playlists/:id/update' },
+    },
```

Comment:
A sub-menu appears under "My Playlist". 
You have to adapt the system to not display this sub-menu.

![Screenshot (131)](https://user-images.githubusercontent.com/25994652/219009145-94ca6353-3038-4b9c-837e-25e6340bc51f.png)

```suggestion
    showSubRoutes: false,
    subRoutes: {
      UPDATE_PLAYLIST: { path: '/my-playlists/:id/update' },
    },
```